doc/index-tex.rst

```diff
@@ -0,0 +1,24 @@
+:orphan:
+
+..
+    Zephyr Project documentation main file
+
+.. _zephyr-home-tex:
+
+Zephyr Project Documentation
+############################
+
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Contents
+
+   introduction/index.rst
+   getting_started/index.rst
+   contribute/index.rst
+   development_process/index.rst
+   guides/build/index.rst
+   application/index.rst
+   reference/index.rst
+   guides/index.rst
+   security/index.rst
```

Comment:
I'm not a fan of having two index files, we'll need to maintain 2 duplicate toctrees this way. I also don't see why we can't have the Indices and Tables chapter in the latex output.

Reply:
The content is minimal, it is no way the same we maintain in the html variant. If you find a better way to have role based TOC, then I am all ears.

The indices are created automatically, no need to add them. In the html index we are just adding a link to the reference.